app/models/inbox_comment.rb

```diff
@@ -31,4 +31,17 @@ def self.count_unread
     self.count(:conditions => {:read => false})
   end
 
+  # Get only the comments with a feedback_comment that exists
+  def self.with_feedback_comment
+    # Get an array of the ids of inbox comments that have existing feedback_comments 
+    inbox_comments_with_feedback_comment = []
+    self.find_each do |inbox_comment|
+      unless inbox_comment.feedback_comment.nil? || inbox_comment.feedback_comment.is_deleted?
+        inbox_comments_with_feedback_comment << inbox_comment
+      end    
+    end
+     # Turn that array into ActiveRecord Relation objects
+    inbox_comments_with_feedback_comment.map{ |i| i.id }
```

Comment:
Space missing to the left of {.<br>Pass `&:id` as an argument to `map` instead of a block.